example/full/table.properties

```diff
@@ -152,6 +152,11 @@ sleeper.table.compaction.job.id.assignment.commit.async=true
 # instance property.
 sleeper.table.compaction.job.commit.async=true
 
+# If true, asynchronous commits will be batched if they are enabled. The commits will be gathered into
+# a larger transaction per Sleeper table. If false, the commits will sent directly to the commit
+# lambda. 
```

Comment:
This would be clearer as:

"This property affects whether commits of compaction jobs are batched before being sent to the state store commit queue to be applied by the committer lambda. If this property is true and asynchronous commits are enabled then commits of compactions will be batched. If this property is false and asynchronous commits are enabled then commits of compactions will not be batched and will be sent directly to the committer lambda."